tests/tasks/sanity-tests.yaml

```diff
@@ -0,0 +1,11 @@
+- name: "Run sanity tests for {{ _collection_name }}"
+  vars:
+    _collection_name: >-
+      {%- set _namespace = _collection_directory.split('/')[-2] -%}
+      {%- set _name = _collection_directory.split('/')[-1] -%}
+      {{ _namespace }}.{{ _name }}
+  command: ansible-test sanity --skip-test pylint --docker
```

Comment:
done

Reply:
Because ansible-test runs from within an ansible-playbook, the colors are eaten by the CLI so it looks like this instead: ``\u001b[0m\n\u001b[31mERROR: plugins/modules/activation_key.py:337:23: SyntaxError: for product in product_content['results']``.

I'll take it out of this particular implementation so the output is a bit cleaner.